scripts/error-codes/codes.json

```diff
@@ -280,7 +280,7 @@
   "281": "Finished root should have a work-in-progress. This error is likely caused by a bug in React. Please file an issue.",
   "282": "If the root does not have an updateQueue, we should have already bailed out. This error is likely caused by a bug in React. Please file an issue.",
   "283": "Element type is invalid. Received a promise that resolves to: %s. Promise elements must resolve to a class or function.",
-  "284": "Expected ref to be a function, a string, an object returned by React.createRef(), or null.",
```

Comment:
Can we give this a new error code somehow so that older versions of React decode an error message with their respective ref type?

Reply:
I did it this way intentionally for log continuity. The new message isn't incorrect for older versions, per se. It doesn't list "string" as a valid type but that's fine because we'd rather then use one of the other ones anyway.

Reply:
That said I also think it's fine if we make a new error code. Generally I try to avoid creating new ones if the change isn't significant, though.